src/language/providers/intellisense/attributeHoverItems.ts

```diff
@@ -0,0 +1,339 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License Version 2.0
+ * (the "License"; you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// prettier-ignore
+
+export function attributeHoverValues(attributeName: string): string {
+  let hoverValue: string = ''
+  switch (attributeName) {
+    case 'name':
+      hoverValue = 'specify name'
+      break
+    case 'ref':
+      hoverValue = 'Specifies the name of an element in this schema'
+      break
+    case 'minOccurs':
+      hoverValue = 'Minimum number of times element will occur'
+      break
+    case 'maxOccurs':
+      hoverValue = 'Maximum number of times element will occur'
+      break
+    case 'dfdl:occursCount':
+    hoverValue = 'dfdl:occursCount property takes an expression which commonly looks in the Infoset via an expression to obtain the count from another element.'
+    break

```

Comment:
Fix indent.  Run `yarn lint:fix`.

Reply:
Replaced "hoverValue=" and break; with return of the string.

Reply:
When ""plugins": ["prettier-plugin-svelte"]," was added to the prettierrc file, prettier stopped working for typescript. At least on my machine. I removed that line from the prettierrc on my machine and prettier executed on the ts files that were reported as having format problems.

Reply:
when I run yarn prettier fix -c, I get:
[warn] Code style issues found in 70 files. Run Prettier to fix.
when I run yarn prettier fix, it doesn't change anything.
VS Code show that prettier ran also
![image](https://github.com/apache/daffodil-vscode/assets/72815523/a17c7583-6fff-48e6-9934-813e6d230518)

Reply:
@rthomas320 Don't run `yarn prettier fix`, run `yarn lint:fix`. You may be running a global version of `prettier` instead of the one for `daffodil-vscode` specifically. I also think `yarn prettier fix` is not a command I believe if you want to run the fix manually instead of using `yarn lint:fix` it would be `yarn prettier src -w`, `src` is the directory to look at and `-w` tells prettier to fix the issues it finds.

Reply:
Once all the CI is passing I will add a +1

Reply:
I checked-out the .prettierrc and prettierignore files from main and ran yarn lint:fix.  That fixed the CI check format issues.